examples/step-79/doc/intro.dox

```diff
@@ -267,50 +267,50 @@ The solution of the optimization problem then needs to satisfy what are known as
   functions -- in other words, all of these conditions have to be stated in weak
    form as is typically the basis for finite element methods anyway.
 
-The barrier method allows us to initially weaken the ``complementary slackness"
+The barrier method allows us to initially weaken the "complementary slackness"
 as required by the typical KKT conditions. Typically, we would require that
 $s_i z_i = 0$, but the barrier formulations give KKT conditions where
 $s_i z_i = \alpha$, where $\alpha$ is our barrier parameter. As part of the
 barrier method, this parameter must be driven close to 0 to give a good
 approximation of the original problem.
 
 In the following, let us state all of these conditions where
-$d_{\{\cdot\}}$ is a test function that is naturally paired with variational
-derivatives of the Lagrangian with respect to the $\{\cdot\}$ function.
+$d_{\{\bullet\}}$ is a test function that is naturally paired with variational
```

Comment:
I think that the `\bullet` is bigger than `\cdot` and should, therefore, be more easily readable.

Reply:
Took me looking back and forth a couple times, but I agree, this is a lot easier to see as a standalone dot.